client/payments/data/reducer.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Internal dependencies
+ */
+import TYPES from './action-types';
+
+const DEFAULT_STATE = {
+	paymentGateways: [],
+};
+
+const reducer = ( state = DEFAULT_STATE, action ) => {
```

Comment:
We also at minimum need some unit tests on the reducer, as a stretch goal I would like to see some tests of the generators (resolvers), but I understand that can be tough, so no pressure.

Sorry to pile in with so many comments, you've done a great job of this, just a few things to consider.

Reply:
no worries :) all valid comments. Thank you for the suggestions 👍

Reply:
Hey @samueljseay I think I've covered all your suggestions. Could you please check the [new files](https://github.com/woocommerce/woocommerce-admin/pull/6918/files) (ts files)? Thank you!